fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -295,6 +342,21 @@ if [ ! -e "$fMRIFolder" ] ; then
 fi
 cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
 
+# --- Do slice time correction if indicated
```

Comment:
Can we change two lines prior to:
`${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"`

Reply:
done.

Reply:
Comparing the diffs, looks l may have put my comment in the wrong place.
The line was previously:
`cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz`
so the replacement would be:
`${FSLDIR}/bin/imcp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName"`

Reply:
Fixed.